apps/backend/src/schemas/cache.py

```
@@ -0,0 +1,541 @@
+"""
+Cache metadata Pydantic schemas for request/response validation.
+"""
+
+from datetime import datetime
+from typing import Any, List, Optional
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)